### DISCUSSION TITLE
Accessing E00 Late in Semester

### DISCUSSION THREAD

**Discussion by leharris3** [comp426-2023-spring/syllabus#20] [General]
A few other students and I began committing assignments to the Fall 2022 Repo for this class instead of the current version. For those of us who took the Qualtrics survey at the beginning of the year but did not upload it to the Spring E00 Repo, is there any way to get a GitHub classroom link this late in the semester? Thank you!

**Comment by jdmar3** [comment]
I think we spoke via email about this, but for others, if you filled out the preliminary exam, it should have emailed you the list of your responses including a link at the end to the GitHub classroom invite link for E00. That will create the repo.
